src/command.c

```diff
@@ -989,3 +1004,121 @@ void dv_run_ipu_command(struct dmp_cmb *cmb, void *bar_logical)
 	}
 	iowrite32(0x1, REG_IO_ADDR(bar_logical, 0x02a0));
 }
+
+// If already a command is converted and not executed yet, -EBUSY is returned.
+static int dv_convert_maximizer_v0(struct device *dev, struct dmp_cmb *cmb,
+			struct dmp_dv_kcmdraw __user *user_cmd, size_t size)
+{
+	struct dmp_cmb_list_entry *cmb_node;
+	struct dmp_dv_kcmdraw_maximizer_v0 cmd;
+	size_t cmd_size = sizeof(uint32_t) * 2 * 8;
+	uint32_t *cmd_buf;
+	uint32_t in_base_addr = 0, in_sz;
+	uint32_t out_base_addr = 0, out_sz;
+	uint32_t num_pixel;
+	uint32_t block_size;
+	uint32_t last_block_src;
+	int i = 0;
+	int ret;
+
+	cmb_node = list_first_entry(&cmb->cmb_list,
+			struct dmp_cmb_list_entry, list_node);
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket